core/cap-0056.md

```diff
@@ -139,33 +125,19 @@ index 49d1c3c77..7665f16bf 100644
 +    LOG_ON_ERROR = 2
 +};
 +
-+struct Index
-+{
-+    SCSymbol label;
-+    SCVal value; //Cannot be SCO_VEC or SCO_MAP. If SCO_BINARY, size of bin is limited to 64 bytes
-+}
-+
-+enum ContractLogBodyType
-+{
-+    SCVAL_AND_INDEX = 0
-+};
-+
-+union ContractLogBody switch (ContractLogBodyType type)
-+{
-+case SCVAL_AND_INDEX:
-+    struct
-+    {
-+        Index indexes<5>;
-+        SCVal body;
-+    } SCValAndIndex;
-+}
-+
 +struct ContractLog
 +{
 +    Hash contractID;
-+    SCSymbol functionName;
 +    LogType type;
-+    ContractLogBody body;
++    SCVal body;
++
++    // reserved for future use
++    union switch (int v)
++    {
++    case 0:
++        void;
++    }
++    ext;
```

Comment:
I think you can use `ExtensionPoint` which lives in `Stellar-types.x`, and I get the feeling someone, maybe @stanford-scs, asked us to start putting it at the start of structs for some reason, but I don't remember why. The latest account extension was done this way.